### PR TITLE
Clean up package.jsons of SDK-s & Update SDK-Core's Subgraph schema & Clean up SDK-Redux demo project

### PR DESCRIPTION
**Pull request by kasparkallas**
* move dependency to devDependency (correct place)
* remove unused scripts and packages
* make react/redux mandatory peerDependency for now
* add linting to sdk-core project
* Clean-up SDK-Redux demo project
* Update Subgraph schema for SDK-Core

### CODE REVIEW

File: packages/sdk-redux/package.json [hunk 1]
```diff
@@ -38,28 +33,24 @@
...
-        "open-cli": "^6.0.1",
-        "prettier": "^2.1.1",
-        "ts-node": "^9.0.0",
+        "prettier": "^2.5.1",
+        "ts-node": "^10.4.0",
         "typedoc": "^0.22.10",
-        "typescript": "^4.3.5"
+        "typescript": "^4.5.2"
+    },
+    "peerDependencies": {
```
Comment (kasparkallas) [inline packages/sdk-redux/package.json:51]: I'd leave React as a mandatory peer dependency until we have a working example up with another framework.

File: packages/sdk-core/package.json [hunk 2]
```diff
@@ -21,10 +21,9 @@
         "posttest": "yarn testenv:stop",
         "check-updates": "ncu --target minor --dep prod,dev",
         "doc:html": "typedoc --out build/docs --entryPoints ./src/index.ts",
-        "fix": "run-s fix:*",
-        "fix:prettier": "prettier \"src/**/*.ts\" --write",
-        "fix:lint": "eslint src --ext .ts --fix",
-        "lint": "echo linting",
```
Comment (kasparkallas) [inline packages/sdk-core/package.json:27]: Didn't lint before.

File: packages/sdk-core/.eslintrc.json [hunk 3]
```diff
@@ -1,10 +1,57 @@
 {
```
Comment (kasparkallas) [inline packages/sdk-core/.eslintrc.json:1]: This configuration was not proper for TypeScript previously. I copied the one from sdk-redux here (which was initially created by scaffolding a TS library).

File: packages/sdk-core/package.json (hunk 2 above)
Comment (0xdavinchee) [reply packages/sdk-core/package.json:27]: nice

**Review by 0xdavinchee — APPROVED** [review]
gmi

**Comment by kasparkallas** [comment]
Clean-up demo app here: https://sdk-redux.netlify.app/

**Review by 0xdavinchee — APPROVED** [review]
gmi

**Pull request merged**